blacklist.txt

```diff
@@ -0,0 +1,9 @@
+fun:*roc_sender_connect*
```

Comment:
Can we isolate UB in a few helper conversion/validation functions, similar to proto_from_user, and blacklist only them? I'd prefer to avoid disabling sanitizers for the whole API function.

Also, is it possible to disable only specific sanitizers? 

Also, let's move this file to src/ and rename to something like sanitizer_blacklist.txt

Reply:
> Can we isolate UB in a few helper conversion/validation functions, similar to proto_from_user, and blacklist only them? I'd prefer to avoid disabling sanitizers for the whole API function.

No. After the test runner passes invalid value for enum argument even accessing enum variable in caller causes runtime error.

> Also, is it possible to disable only specific sanitizers?

Yes, It's possible and I have updated it.

> Also, let's move this file to src/ and rename to something like sanitizer_blacklist.txt

Done.

Reply:
> No. After the test runner passes invalid value for enum argument even accessing enum variable in caller causes runtime error.

I see. I checked it out and found a workaround: we can change signature of, for example, `interface_from_user()` from `roc_interface in` to `const roc_interface& in`. In this case `roc_sender_*()` functions that invoke `interface_from_user()` may be removed from blacklist. Same should work for `proto_from_user()`.